components/mosart/bld/namelist_files/namelist_defaults_mosart.xml

```diff
@@ -32,4 +39,34 @@ for the CLM data in the CESM distribution
 <frivinp_rtm rof_grid="r05" >rof/mosart/MOSART_global_half_20180721a.nc</frivinp_rtm>
 <frivinp_rtm rof_grid="NLDAS" >rof/mosart/MOSART_NLDAS_8th.nc</frivinp_rtm>
 
+<!-- WRM -->
+
+<paraFile rof_grid="NLDAS" >rof/mosart/US_reservoir_8th_NLDAS3_c20161220_updated_20170314.nc</paraFile>
```

Comment:
Has this file been added to e3sm inputdata sever?

Reply:
I don't think I have the permission to add this file to the input data server. Could you help me uploading it? You can find it in Constance input data directory.